toplevel/metasyntax.ml

```diff
@@ -991,8 +992,8 @@ let compute_syntax_data df modifiers =
   let symbols' = remove_curly_brackets symbols in
   let need_squash = not (List.equal Notation.symbol_eq symbols symbols') in
   let ntn_for_grammar = make_notation_key symbols' in
-  check_rule_productivity symbols';
-  let msgs,n = find_precedence n etyps symbols' in
+  if not onlyprint then check_rule_productivity symbols';
+  let msgs,n = if onlyprint then ([],0) else find_precedence n etyps symbols' in
```

Comment:
I have no idea.^^ The values should not matter at all because the notation will not be registered with the parser.